lang/gtext_fre.po

```diff
@@ -6849,37 +6849,37 @@ msgstr "Piège Sentinelle"
 #: guitext:985
 msgctxt "Trap names"
 msgid "Mimic Trap"
-msgstr "Piège Mimique"
+msgstr "Piège Copieur"
 
 #: guitext:986
 msgctxt "Trap names"
 msgid "Spawn Trap"
-msgstr "Piège à Invocation"
+msgstr "Piège Invocateur"
 
 #: guitext:987
 msgctxt "Trap names"
 msgid "Wind Trap"
-msgstr "Piège à Vent"
+msgstr "Piège Soufflant"
 
 #: guitext:988
 msgctxt "Trap names"
 msgid "Spitfire Trap"
-msgstr "Piège de Feu"
+msgstr "Piège Lance-flamme"
```

Comment:
In my opinion "Lance-flammes" would be more correct, unless your trap is shooting a single long and powerful flame.